packages/manager/modules/manager-components/src/task-tracker/task-tracker-state.component.js

```diff
@@ -0,0 +1,27 @@
+export default {
+  controller: class {},
+  bindings: {
+    doingLabel: '<',
+    doneLabel: '<',
+    endpoint: '<',
+    errorLabel: '<',
+    heading: '<',
+    interval: '<',
+    onDone: '<',
+    tasks: '<',
+    todoLabel: '<',
+  },
+  template: `
```

Comment:
Why don't we provide directly a component here ? Any benefits of using a compileProvider ?